lib/open_food_network/sales_tax_report.rb

```diff
@@ -9,9 +9,16 @@ def initialize(user, params)
     end
 
     def header
-      ["Order number", "Date", "Items", "Items total (#{currency_symbol})", "Taxable Items Total (#{currency_symbol})",
-        "Sales Tax (#{currency_symbol})", "Delivery Charge (#{currency_symbol})", "Tax on Delivery (#{currency_symbol})", "Tax on Fees (#{currency_symbol})",
-        "Total Tax (#{currency_symbol})", "Customer", "Distributor"]
+      case params[:report_type]
+      when "tax_types"
+        ["Order number", "Date", "Items", "Items total (#{currency_symbol})", "Taxable Items Total (#{currency_symbol})",
+          "Sales Tax (#{currency_symbol})", "Delivery Charge (#{currency_symbol})", "Tax on Delivery (#{currency_symbol})", "Tax on Fees (#{currency_symbol})",
+          "Total Tax (#{currency_symbol})", "Customer", "Distributor"]
+      else
+        ["Order number", "Total excl. VAT"] +
+          relevant_rates.map { |rate| "%s (%.1f%%)" % [rate.name, rate.amount.to_f * 100] } +
```

Comment:
That's right, perhaps you'd like that I not have the "tax by tax rates" report be the default one and instead have the original one ("tax by tax types")?